javascript/javascript1/week2/fullnameformal.js

```diff
@@ -0,0 +1,34 @@
+function getFullname (firstname, surname, useFormalName, gender){
+if (useFormalName) { 
+    if (gender==="man") {
+    console.log(`lord ${firstname} ${surname}`)   
+} else if(gender==="woman") {
+    console.log(`lady ${firstname} ${surname}`)
+}else{
+    console.log(`${firstname} ${surname}`)
+}   
+}
```

Comment:
Please use Prettier and `;`

Reply:
by prettier you mean I need to use the command format?

Reply:
You should format your code. I think you install Prettier during the first HTML/CSS class, and therefore I am suggesting using Prettier. Good article https://www.codementor.io/@myogeshchavan97/how-to-automatically-format-code-in-visual-studio-code-using-prettier-1nebhfbxak

Reply:
Yes I did I think its not working on my computer, not sure why !